helix-term/src/config.rs

```diff
@@ -87,11 +87,9 @@ impl Config {
         root_config: &Config,
         config_path: std::path::PathBuf,
     ) -> Option<toml::Value> {
-        if !config_path.exists() {
-            return None;
-        }
-        if config_path == helix_loader::config_file()
-            || root_config.editor.security.load_local_config
+        if config_path.exists()
+            && (config_path == helix_loader::config_file()
+                || root_config.editor.security.load_local_config)
```

Comment:
Still same thing, this 3 can just `return None`; to reduce one indentation level.

Reply:
...embarrasing, thanks!